ui/src/Error.elm

```diff
@@ -25,4 +25,4 @@ handle error =
             "Bad status: " ++ String.fromInt code
 
         Http.BadBody err ->
-            err
+            "Invalid response body"
```

Comment:
Is the error however printed in a browser console, so we can debug problems?

Reply:
No, but that's a good idea so I print it in console in new commit 4a5f929837a27054ab33b264373a47dc480a3d69 
Because you can't have any side-effects in Elm you will see that I have to do a bit of a trick for logging values that I don't want to return.

Reply:
We'll need to have some kind of better developed failure reporting mechanism to user. For now, user can inspect browser console and MF console messages. LGTM